### ISSUE TITLE
WCAG - Altinn 3 apper

### ISSUE THREAD

**Issue by olavflar** [Altinn/app-frontend-react#2792]
Apper som kan testes:

https://ttd.apps.tt02.altinn.no/ttd/frontend-test#/

Noen av de appene ble testet sist:

https://ssb.apps.tt02.altinn.no/ssb/ra1601-01/
https://ssb.apps.tt02.altinn.no/ssb/ra0244-01/
https://ssb.apps.tt02.altinn.no/ssb/ra0574-01/

https://altinn.studio/repos/srf/rettferdsvederlag
https://altinn.studio/repos/srf/bli-verge
https://altinn.studio/repos/srf/fufinn-behovsendring
https://altinn.studio/repos/srf/fufinn-behovskartlegging
https://altinn.studio/repos/srf/klage-pa-vedtak
https://altinn.studio/repos/srf/legeerklaering
https://altinn.studio/repos/srf/melding-til-statsforvalteren
https://altinn.studio/repos/srf/vergegodtgjoring

https://altinn.studio/repos/dsb/brannaarsak

https://krt.apps.tt02.altinn.no/krt/krt-3000a-1/

For å teste i år, så har jeg også gitt beskjed i Altinn-slacken for å få flere apper å kunne teste med

**Comment by olavflar** [comment]
[Hjelpdokument til testing for tilgjengelighetserklæring.xlsx](https://github.com/user-attachments/files/18039069/Hjelpdokument.til.testing.for.tilgjengelighetserklaering.xlsx)

Last ned denne og lag en kopi med navn for hvilken app som testes
